x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -64,47 +64,6 @@ const RiskScoreErrorPanel = ({ errors }: { errors: string[] }) => (
           ))}
         </>
       </EuiAccordion>
-
-      <EuiAccordion id="risk-engine-privileges" buttonContent={i18n.CHECK_PRIVILEGES}>
```

Comment:
This accordion in our error messages has been superseded by the privileges callout at the top of the page. Hopefully these new UI restrictions will mean the user wont get these privilege errors anyway.